cli/cmd/service/token/issue.go

```diff
@@ -0,0 +1,53 @@
+package token
+
+import (
+	"fmt"
+
+	"github.com/rilldata/rill/cli/pkg/cmdutil"
+	"github.com/rilldata/rill/cli/pkg/config"
+	"github.com/rilldata/rill/cli/pkg/dotrill"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"github.com/spf13/cobra"
+)
+
+func IssueCmd(cfg *config.Config) *cobra.Command {
+	var name string
+	issueCmd := &cobra.Command{
+		Use:   "issue [service-name]",
+		Args:  cobra.NoArgs,
+		Short: "Issue service token",
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Probably we should use the positional arg, like in the `rill service` commands? Or allow both flag and positional?

Reply:
If we keep both, then if nothing is provided then it will take the default value of flag and result accordingly, keeping both for will update as per user experience